Gifski/IntTextField.swift

```diff
@@ -0,0 +1,61 @@
+import AppKit
+import Carbon
+
+final class IntTextField: NSTextField, NSTextFieldDelegate {
+	// Delta used for arrow navigation
+	var delta = 1
+
+	// Delta used for option + arrow navigation
+	var biggerDelta = 10
+
+	var onBlur: ((Int) -> Void)?
+	var onTextDidChange: ((Int) -> Void)?
+
+	required init?(coder: NSCoder) {
+		super.init(coder: coder)
+		setup()
+	}
+
+	override init(frame frameRect: NSRect) {
+		super.init(frame: frameRect)
+		setup()
+	}
+
+	private func setup() {
+		delegate = self
+	}
+
+	override func performKeyEquivalent(with event: NSEvent) -> Bool {
+		let key = Int(event.keyCode)
```

Comment:
I think you can use https://developer.apple.com/documentation/appkit/nsevent/3019455-specialkey for this instead.

Reply:
Oh nice didn't notice that! Thanks, updated 👍